src/main/java/org/project_kessel/relations/client/HealthClient.java

```diff
@@ -0,0 +1,99 @@
+package org.project_kessel.relations.client;
+
+import java.util.logging.Logger;
+
+import io.grpc.Channel;
+import io.grpc.stub.StreamObserver;
+import io.grpc.stub.StreamObservers;
+import io.smallrye.mutiny.operators.multi.processors.UnicastProcessor;
+
+import org.project_kessel.api.relations.v1.GetLivezReply;
+import org.project_kessel.api.relations.v1.GetLivezRequest;
+import org.project_kessel.api.relations.v1.GetReadyzReply;
+import org.project_kessel.api.relations.v1.GetReadyzRequest;
+import org.project_kessel.api.relations.v1.KesselHealthGrpc;
+import io.smallrye.mutiny.Uni;
+
+public class HealthClient {
+    private static final Logger logger = Logger.getLogger(CheckClient.class.getName());
```

Comment:
This needs to be HealthClient.class.getName()

(Even though the logger is not used yet.)